apps/utils/locust_tests/lambda/server-load/broker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM public.ecr.aws/lambda/python:3.8 as base
+RUN  yum install git gcc python3-devel -y
+
+FROM base as python
+# Install the function's dependencies using file requirements.txt
+# from your project folder.
+COPY requirements.txt .
+RUN  pip3 install -r requirements.txt
+ENV  PATH="/var/lang/lib/python3.8/site-packages:${PATH}"
+
+FROM python as app
```

Comment:
doing multistage docker builds like this just adds bloat to the final runtime image 
Snyk has a good [example](https://snyk.io/blog/best-practices-containerizing-python-docker/) on how to approach multistage python builds 
```
FROM python:3.10-slim as build
RUN apt-get update
RUN apt-get install -y --no-install-recommends \
build-essential gcc 

WORKDIR /usr/app
RUN python -m venv /usr/app/venv
ENV PATH="/usr/app/venv/bin:$PATH"

COPY requirements.txt .
RUN pip install -r requirements.txt

FROM python:3.10-slim@sha256:2bac43769ace90ebd3ad83e5392295e25dfc58e58543d3ab326c3330b505283d
WORKDIR /usr/app/venv
COPY --from=build /usr/app/venv ./venv
COPY . .

ENV PATH="/usr/app/venv/bin:$PATH"
CMD [ "python", "app.py" ]
```